gnssanalysis/gn_io/igslog.py

```diff
@@ -101,6 +129,22 @@ def find_recent_logs(logs_glob_path: str, rnx_glob_path=None) -> _pd.DataFrame:
     return recent_logs_df
 
 
+def determine_log_version(data: bytes):
+    REGEX_VERSION_1 = _re.compile(rb"""(site log\))""")
```

Comment:
REGEX_VERSION_1 should be _REGEX_VERSION_1 to follow the existing coding standard

Reply:
Fixed.
And yep, there is a closed bracket in version 1 - I guess they weren't expecting it to change